aspnetcore/blazor/debug.md

```diff
@@ -15,28 +15,27 @@ uid: blazor/debug
 
 [!INCLUDE[](~/includes/blazorwasm-preview-notice.md)]
 
-*Early* support exists for debugging Blazor WebAssembly using the browser dev tools in Chromium-based browsers (Chrome/Edge). Work is in progress to:
+As of ASP.NET Core 3.2 Preview 3, Blazor WebAssembly apps can be debugged using the browser dev tools in Chromium-based browsers (Chrome/Edge) in Visual Studio and Visual Studio Code.
 
-* Fully enable debugging in Visual Studio.
-* Enable debugging in Visual Studio Code.
-
-Debugger capabilities are limited. Available scenarios include:
+Available scenarios include:
 
 * Set and remove breakpoints.
-* Single-step (`F10`) through the code or resume (`F8`) code execution.
-* In the *Locals* display, observe the values of any local variables of type `int`, `string`, and `bool`.
+* Run the app with debugging support in Visual Studio (<kbd>F5</kbd> support).
+* Single-step (<kbd>F10</kbd>) through the code or resume (<kbd>F8</kbd>) code execution.
```

Comment:
F8 is only used for resuming execution in a browser. In VS or VS Code you'd use F5.